lib/workers/repository/process/fingerprint-fields.ts

```diff
@@ -0,0 +1,37 @@
+import type { UpdateType } from '../../../config/types';
+
+export interface UpgradeFingerprintConfig {
+  manager: string;
+  packageFile?: string;
+  lockFile?: string;
+  currentDigestShort?: string;
+  datasource?: string;
+  digestOneAndOnly?: boolean;
+  fixedVersion?: string;
+  currentVersion?: string;
+  lockedVersion?: string | null;
+  newVersion?: string;
+  newDigest?: string;
+  newName?: string;
+  updateType?: UpdateType;
+  replaceString?: string;
+  extractVersion?: string;
```

Comment:
Are you sure `extractVersion` is used during update? I thought only during lookup

Reply:
Not sure. I was just not sure about the difference between extractVersion and currentVersion so included it so we can discuss/